wasm/jit/jit_arm64.go

```diff
@@ -327,16 +356,87 @@ func (c *arm64Compiler) compileCallIndirect(o *wazeroir.OperationCallIndirect) e
 	return fmt.Errorf("TODO: unsupported on arm64")
 }
 
+// compileDrop implements compiler.compileDrop for the arm64 architecture.
 func (c *arm64Compiler) compileDrop(o *wazeroir.OperationDrop) error {
-	return fmt.Errorf("TODO: unsupported on arm64")
+	r := o.Range
+	if r == nil {
+		return nil
+	} else if r.Start == 0 {
+		// If the drop starts from the top, we just mark all the regiters
```

Comment:
nit: "just " is almost always deletable

Reply:
reword, and insert the why for the what
"When the drop starts from the top the stack, mark all registers unused because XXX"